### PR TITLE
Thank you presenters + resources addition

### PR DESCRIPTION
**Pull request by SophiaL9358**
The style.css file in Homepage might be a bit wonky because I ctrl c + v'ed together the css generated by the Watch SASS and the file's original css.

### CODE REVIEW

**Review by InternetRamen — CHANGES_REQUESTED** [review]

**Comment by InternetRamen** [comment]
Everything else looks good! I changed `p-5` to `px-sm-5 py-5` so the buttons don't get cut off on mobile. I think we could delete og.txt and copy.txt. Thank you so much!

File: Homepage/css/copy.txt
[the remark this answers is not in the document]
Comment (SophiaL9358) [reply Homepage/css/copy.txt:1]: Should be deleted now

File: Homepage/css/og.txt
[the remark this answers is not in the document]
Comment (SophiaL9358) [reply Homepage/css/og.txt:1]: Should be deleted now  x2

**Review by InternetRamen — APPROVED** [review]
LGTM Thanks for submitting! Welcome to the team

**Pull request merged**